lib/synapse/version.rb

```diff
@@ -1,3 +1,3 @@
 module Synapse
-  VERSION = "0.13.8"
+  VERSION = "0.13.9"
```

Comment:
don't bump the version as part of this PR. we should test this separately, and if it works then we can bump the version number as a separate change and then release the new version